README.md

```diff
@@ -1,32 +1,127 @@
+<h1 align="center">
+  <br>
+  <a href="https://github.com/Mudlet/Mudlet"><img src="https://bazaar.launchpad.net/~mudlet-makers/mudlet/artwork/download/head:/mudlet_main_256px.pn-20171109060211-hidv2woj9dffwyql-11/mudlet_main_256px.png" alt="Mudlet"></a>
+  <br>
+  Mudlet
+  <br>
+</h1>
+
+<h4 align="center">Play immersive, multiplayer, pure-text RPGs on Mudlet.</h4>
+
+<p align="center">
+  <a href="http://mudlet.org/download/">
+    <img src="https://travis-ci.org/Mudlet/Mudlet.svg?branch=development"
+         alt="Build status">
+  </a>
+  <a href="https://discord.gg/kuYvMQ9">
+    <img src="https://img.shields.io/badge/discord-join%20chat-7289DA.svg">
+  </a>
+  <a href="https://github.com/mudlet/mudlet/issues">
+    <img src="https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat">
+  </a>
+  <a href="https://github.com/Mudlet/Mudlet/stargazers">
+    <img src="https://img.shields.io/github/stars/Mudlet/Mudlet.svg"/>
+  </a>
+</p>
+
+<p align="center">
+  <a href="#about">About</a> •
+  <a href="#key-features">Key Features</a> •
+  <a href="#how-to-use">How To Use</a> •
+  <a href="#download">Download</a> •
+  <a href="#credits">Credits</a> •
+  <a href="#roadmap">Roadmap</a> •
```

Comment:
Credits and Roadmap are ordered here differently than below

Reply:
Fixed.

Reply:
Also, Download needs to go before How To Use as below

Reply:
Okay, adjusted.